packages/react-component-library/src/components/SelectE/SelectE.test.tsx

```diff
@@ -135,6 +139,20 @@ describe('SelectE', () => {
           expect(onChangeSpy).toHaveBeenCalledTimes(1)
           expect(onChangeSpy).toHaveBeenCalledWith('two')
         })
+
+        describe('and the clear button is clicked', () => {
+          beforeEach(() => {
+            wrapper.getByTestId('select-clear-button').click()
+          })
+
+          it('resets the value', () => {
+            expect(wrapper.getByTestId('select-input')).toHaveValue('')
+          })
+
+          it('calls the `onChange` callback', () => {
+            expect(onChangeSpy).toHaveBeenCalledWith(undefined)
```

Comment:
Should we be calling it with `undefined` or `null`?

Would be good to update the type of `onChange` in `SelectEProps` too (currently `onChange?: (value: string) => void`, should be `value?: string` if we're using undefined).